README.md

```diff
@@ -16,7 +16,7 @@ cd build
 cmake -DROBOT_NAME=iCubGenova04 ..
 ```
 
-After this, append `<icub-gazebo-wholebody>/build/gazebo/models` to the `GAZEBO_MODEL_PATH` enviromental variable.
+After this, append `<icub-gazebo-wholebody>/build/gazebo` to the `GAZEBO_MODEL_PATH` enviromental variable.
 
```

Comment:
I'm pretty sure with `.../models` was working. Trusting you in this.

Reply:
For the record, this was due to an issue in the CMakeLists.txt. to be fixed in the next commit